students/WWhite/lesson09/oo_mailroom.py

```diff
@@ -0,0 +1,168 @@
+#!/usr/bin/env python3
+
+# -------------------------------------#
+# Desc: Object Oriented Mailroom
+# Dev: Will White
+# Date: 7/2/2018
+# ChangeLog: (When,Who,What)
+# -------------------------------------#
+import math
+
+
+class Donor(object):
```

Comment:
note that in python3 you don't need to extend from object -> `class Donor:` is sufficient